junit/src/test/scala/scala/collection/parallel/TaskTest.scala

```diff
@@ -48,4 +53,43 @@ class TaskTest {
     val r = c.filter(_ != 0).map(_ + 1)
     assertSame(myTs, r.tasksupport)
   }
+
+  // was: Wrong exception: expected scala.collection.parallel.TaskTest$SpecialControl$1 but was java.lang.IllegalArgumentException
```

Comment:
I don't get this comment

Reply:
Before the fix, the `assertThrows` fails with this message.